src/components/mobile/ConfirmTransactionSignModal.vue

```diff
@@ -60,7 +60,7 @@
 
 <script>
 import { mapState } from 'vuex';
-import { SCHEMA } from '@aeternity/aepp-sdk';
+import { Tag } from '@aeternity/aepp-sdk-next';
```

Comment:
why next?

Reply:
I would rename it back to `@aeternity/aepp-sdk` after finishing the update